### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by andrew-me** [auth0/lock#1469]
Lock version: 11.3

It'd be great to get Lock meeting the first [level of accessibility recommended by WAI](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&currentsidebar=%23col_customize&levels=aa%2Caaa)

I did a quick audit of the login form, here are some suggestions:

**No alt text on images.** 

This is for the logo. It could arguably be an empty alt tag, indicating to screen readers that it's a decorative element.

**The page does not contain a heading, skip link, or landmark region.** 

Adding ways to bypass repetitive content lets keyboard users navigate the page more efficiently

**Form elements do not have associated labels.**

Helps users with screen readers

** id attributes on the page are not unique**

The value of an id attribute must be unique to prevent other instances from being overlooked by assistive technologies.

The ids in question are on SVGs, so it's possible we could use aria to hide these elements from screen readers instead

** Page specifies valid language**

<html> should have a [lang] attribute

**Keyboard accessible**

No focus indicator on the submit button or password link

**Comment by andrew-me** [comment]
I'm preparing a pull request for this

**Comment by andrew-me** [comment]
Pull request here https://github.com/auth0/lock/pull/1471

**Comment by bigblind** [comment]
The PR was merged, so I think this issue should be closed.
